features/app_update.feature

```diff
@@ -65,6 +71,20 @@ Scenario: Ensure app update can be obtained from HQ
     Then I touch the "Recheck" text
     Then I see the text "Current version: 11"
 
+    # Change the update endpoint to "Latest build" and update again 
+    Then I go back to the home screen
+    Then I select "Settings" menu item
+    Then I touch the "Update Options" text
+    Then I touch the "Latest build" text
+    Then I go back to the home screen
+    Then I select "Update App" menu item
+    Then I wait to see "Update to version"
+    Then I see the text "version 12"
```

Comment:
Should this say `version 19`? Basing that off the latest version on the ui-test domain: https://www.commcarehq.org/a/ui-tests/apps/view/0841e3049fd00508782eb4f996e36611/releases/